SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Supported versions include Red v3 and higher. Red v2 and lower are end of life. And will not receive security updates of any kind. If you are using v2 we recommend you upgrade to v3.
```

Comment:
This doesn't make sense. Red V2 is not the definition of End of Life.
```suggestion
Supported versions include Red v3 and higher. Red v2 and lower are at end of life. And will not receive security updates of any kind. If you are using v2 we recommend you upgrade to v3.
```

Reply:
Kinda disagree here, 
`Red v2 and lower reached end-of-life and will not recive security update or fixes of any kind...` would be what makes sense here.

Reply:
Have reached maybe?

Reply:
`Have reached` would be Present Perfect Tense, and `reached` is Simple Past Tense, due to the fact that they reached EOL prior to the release of this PR, simple past tense would be the appropriate tense to use here

Reply:
Our minimum supported version shoud be 3.1.8 due to the discord.py fix for application crash with animated emoji